src/components/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 import checURI from '@alepop/check-for-uri-handler';
```

Comment:
not specifically your fault, but this should be `checkURI`, probably.

Reply:
ooops